Libs/GObject/GObject.csproj

```diff
@@ -4,6 +4,7 @@
   </ItemGroup>
 
   <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
If you change the target framework please do it the `Libs/Dirctory.Build.props` file as this keeps the settings for all our projects identical.

Reply:
@badcel it's fixed 👍🏾